StRoot/StEventUtilities/StEbyET0.cxx

```diff
@@ -69,17 +69,17 @@ double StEbyET0::getTime(StEvent* event, int mode) {
                 if (info[0] > 0 && info[1] > 0)
                   coordinate = 0.5*(info[0]+info[1]);
                 break;
-      case  5 : // next 4 use east trigger detectors
+      case  4 : // next 4 use east trigger detectors
+      case  5 :
       case  6 :
-      case  7 :
-      case  8 : getTriggerInfo(event,static_cast<trigDetType> (detector-4),info);
```

Comment:
Interesting, so the original code here was able to provide unsupported kVPDeast = 8 - 4 = 12 - 8:
https://github.com/star-bnl/star-sw/blob/94fbc35658cd62fbeefee130575a7d385ac6ea9d/StRoot/StEventUtilities/StEbyET0.h#L17-L29
https://github.com/star-bnl/star-sw/blob/94fbc35658cd62fbeefee130575a7d385ac6ea9d/StRoot/StEventUtilities/StEbyET0.cxx#L143-L162

Reply:
When detector = 4 (same as kVPDeast), then we want to pass kVPD to getTriggerInfo().

Are you concerned about doing arithmetic (i.e. 8 - 4) on the enumeration?

Reply:
I wasn't concerned. But I have a suggestion how to implement that, if you are interested. You could add
```c++
case kVPD:
case kVPDeast:
case kVPDwest:
info[0] = //...
```
to getTriggerInfo instead of that subtraction.

Reply:
Oh, I see - yeah, that would work - thanks for the suggestion, @veprbl . I'll consider doing that in a separate PR that goes into a later library release. For now, I just want to make the minimal change so that we can patch SL23d.